Tasks/VSMobileCenterTest/Strings/resources.resjson/en-US/resources.resjson

```diff
@@ -1,6 +1,6 @@
 {
   "loc.friendlyName": "Mobile Center Test",
-  "loc.helpMarkDown": "Test mobile app packages with Visual Studio Mobile Center.",
+  "loc.helpMarkDown": "[Support Information](https://intercom.help/mobile-center/)",
```

Comment:
It looks like the previous **helpMarkDown** text is just a duplicate of the **description** text.  How about we expand Luke's new text to be more explanatory as follows:

`For help with this task, visit the Visual Studio Mobile Center [support site](https://intercom.help/mobile-center/).`

In the future, we or Mobile Center can include a link to docs that get created for this task.